WordPress/Classes/ViewRelated/Jetpack/Install/View/JetpackRemoteInstallStateView.swift

```diff
@@ -5,6 +5,17 @@ protocol JetpackRemoteInstallStateViewDelegate: AnyObject {
     func customerSupportButtonDidTouch()
 }
 
+struct JetpackRemoteInstallStateViewData {
```

Comment:
Since this is using MVVM to configure the view, what are your thoughts about changing this name to `JetpackRemoteInstallStateViewModel`?

Reply:
That's fine with me; I have no strong preference for it. I was testing out a different approach and the view model name has been used for the previous delegate pattern, so I landed on the "data". 😅 

I'll include this in a clean-up PR later on.